moduleroot/Gemfile

```diff
@@ -2,57 +2,172 @@
 
 source ENV['GEM_SOURCE'] || "https://rubygems.org"
 
-def location_from_env(env, default_location = [])
-  if location = ENV[env]
-    if location =~ /^((?:git|https?)[:@][^#]*)#(.*)/
-      [{ :git => $1, :branch => $2, :require => false }]
-    elsif location =~ /^file:\/\/(.*)/
-      ['>= 0', { :path => File.expand_path($1), :require => false }]
-    else
-      [location, { :require => false }]
-    end
+# Determines what type of gem is requested based on place_or_version.
+def gem_type(place_or_version)
+  if place_or_version =~ /^git:/
+    :git
+  elsif place_or_version =~ /^file:/
+    :file
   else
-    default_location
+    :gem
   end
 end
 
-<%
-def build_location(gem)
```

Comment:
Is this still needed or was that changed based on the ruby-operator, ruby-version removals?